Added `--user` because I got:

```
$ make deps
pip install -r test_requirements.txt
Collecting nose (from -r test_requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/99/4f/13fb671119e65c4dce97c60e67d3fd9e6f7f809f2b307e2611f4701205cb/nose-1.3.7-py2-none-any.whl (154kB)
    100% |████████████████████████████████| 163kB 12.5MB/s
Collecting betamax (from -r test_requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/d1/54/820de0a06d938917103f7d91c024403b9ae521d74e1191fe6f91a70089c9/betamax-0.8.1-py2.py3-none-any.whl
Requirement already satisfied: python-dateutil in /Users/dentarg/Library/Python/2.7/lib/python/site-packages (from -r test_requirements.txt (line 3)) (2.7.5)
Requirement already satisfied: requests>=2.0 in /usr/local/lib/python2.7/site-packages (from betamax->-r test_requirements.txt (line 2)) (2.20.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python2.7/site-packages (from python-dateutil->-r test_requirements.txt (line 3)) (1.10.0)
Requirement already satisfied: urllib3<1.25,>=1.21.1 in /usr/local/lib/python2.7/site-packages (from requests>=2.0->betamax->-r test_requirements.txt (line 2)) (1.23)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /Users/dentarg/Library/Python/2.7/lib/python/site-packages (from requests>=2.0->betamax->-r test_requirements.txt (line 2)) (3.0.4)
Requirement already satisfied: idna<2.8,>=2.5 in /usr/local/lib/python2.7/site-packages (from requests>=2.0->betamax->-r test_requirements.txt (line 2)) (2.7)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python2.7/site-packages (from requests>=2.0->betamax->-r test_requirements.txt (line 2)) (2018.8.24)
Installing collected packages: nose, betamax
Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/usr/local/man'
Consider using the `--user` option or check the permissions.

make: *** [deps] Error 1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,7 +2,10 @@ language: python
 sudo: false
 python:
   - "2.7"
+  - "3.4"
   - "3.5"
+  - "3.6"
+  - "3.7"
 install:
   - make cideps
   - pip install .

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,7 +4,8 @@ python:
   - "2.7"
   - "3.5"
 install:
-  - make deps
+  - make cideps
+  - pip install .
 script:
   - make test
 notifications:

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,4 +1,5 @@
 language: python
+dist: xenial
 sudo: false
 python:
   - "2.7"

--- a/Makefile
+++ b/Makefile
@@ -2,9 +2,12 @@ help:
 	@echo "  deps        install test dependencies"
 	@echo "  test        run tests"
 
-deps:
+cideps:
+	pip install --requirement test_requirements.txt
+
+localdeps:
 	pip install --user --requirement test_requirements.txt
-	pip install .
+	pip install --user --requirement requirements.txt
 
 test:
 	nosetests

--- a/README.md
+++ b/README.md
@@ -77,7 +77,7 @@ In this case documentation will be available at [http://localhost:1234/twingly_s
 
 Install all dependencies
 
-    make deps
+    make localdeps
 
 Run the tests
 

